finance-api/overview.md

```diff
@@ -0,0 +1,19 @@
+Finance API overview
+===
+The Finance API fetches information about transactions that are made through Zettle. For example, card payments made with a Zettle card reader.
+
+# Understand how card payments work at Zettle
+For every merchant who makes transactions with Zettle, they have an Zettle account that handles those transactions.
```

Comment:
This sentence is hard to understand. Who are they for example?

I think we should drop this block since it reveals our internal architecture.